test/utils/test_transforms.py

```diff
@@ -17,7 +17,10 @@
     standardize,
     t_batch_mode_transform,
     unnormalize,
+    _verify_output_shape,
 )
+from botorch.models.model import Model
+from botorch.utils.testing import MockModel, MockPosterior
```

Comment:
oh and can you rearrange these inputs alphabetically? linter will complain about this.

Reply:
What convention do we follow for ordering inputs? Using the notation `from <package> import <module>`, this is currently sorted for the <package> and over <module> for each package. Does linter expect it to be sorted over <module> instead? Trying to get a better understanding for future reference

Reply:
So we have been using iSort so far - I'm about to switch over to usort in #603, but there is one edge case that has to do with FB-infra that needs to be fixed upstream in usort before I can do that. 

Basically to fix this you just need to merge the `botorch.utils.testing` imports with the existing one above and move the models import up to be in alphabetic order.